docusaurus/docs/reference/config-types/_category_.json

```diff
@@ -1,3 +1,3 @@
 {
-  "label": "Builtin Config Types"
+  "label": "Config Types"
```

Comment:
These really are the "Built-in Config Types". I don't think you should change this?

Reply:
The "built-in" qualifier is preserved in the title, in case there's any possibility the reader is confused. This changes the sidebar label to be shorter because it was unnecessarily long and unnecessarily qualified for a sidebar label.